src/Pokemon.java

```diff
@@ -0,0 +1,44 @@
+package src;
+
+public abstract class Pokemon {
+    private String name;
+    private int level;
+    int hp;
```

Comment:
Het is best practice om alle properties private te maken.

Anders als je een pokemon aanmaakt kun je ook een waarde zetten dmv:

pokemon.hp = 10;

terwijl je de waardes van de properties wilt beheren met je constructor en setters.

Dit lijkt in een kleine applicatie niet zo belangrijk, maar als je in een groot team werkt of een applicatie bouwt voor een externe gebruiken wil je 100% owner ship hebben over je properties en het toewijzen van waardes.